apps/prairielearn/src/ee/lib/lti13.ts

```diff
@@ -606,11 +606,11 @@ export async function linkAssessment(
 export function findValueByKey(obj: unknown, targetKey: string): unknown {
```

Comment:
Yuck, I hate this function. Looking at `lti13.test.ts` it also looks like overkill and we could have just explicitly tested for the Canvas error format. This feels a bit risky at this point, however, so it's fine to leave this as is.